### PR TITLE
Mp starting on UI

### PR DESCRIPTION
**Pull request by Ritumka**

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -6,14 +6,26 @@
...
+    <div class="container">
+      <% if notice.present? %>
+        <%= render partial: "shared/flash", locals: { message: notice, css_class: "success" } %>
+      <% end %>
+
+      <% if alert.present? %>
+        <%= render partial: "shared/flash", locals: { message: alert, css_class: "danger" } %>
+      <% end %>
+
+      <%= yield %>
```
Comment (aldoportillo) [inline app/views/layouts/application.html.erb:27]: Good Job

File: config/routes.rb [hunk 2]
```diff
@@ -1,12 +1,13 @@
 Rails.application.routes.draw do
-  resources :likes
-  resources :follow_requests
+  root "photos#index"
```
Comment (aldoportillo) [inline config/routes.rb:2]: Nice

File: lib/tasks/dev.rake [hunk 3]
```diff
@@ -13,6 +13,20 @@ task sample_data: :environment do
     User.destroy_all
   end
 
+  usernames = Array.new { Faker::Name.first_name }
+
+  usernames << "alice"
```
Comment (aldoportillo) [inline lib/tasks/dev.rake:18]: good job

**Review by aldoportillo — APPROVED** [review]
Great job only changing what was relevant to this pull request.

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (viviancreates) [reply app/views/layouts/application.html.erb:27]: Good partials

File: app/views/shared/_navbar.html.erb
Comment (viviancreates) [inline app/views/shared/_navbar.html.erb:1]: Looks good

File: config/routes.rb
Comment (viviancreates) [inline config/routes.rb:1]: Good job changing the order

**Review by viviancreates — APPROVED** [review]
Looks good

**Pull request merged**